tests/e2e/specs/admin/disputes.spec.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import { StoreOwnerFlow } from '../../utils';
+
+const WCPAY_DEPOSITS =
+	config.get( 'url' ) +
+	'wp-admin/admin.php?page=wc-admin&path=/payments/disputes';
+
+describe( 'Admin disputes', () => {
+	beforeAll( async () => {
+		await StoreOwnerFlow.login();
+	} );
+
+	it( 'loads disputes', async () => {
```

Comment:
Perhaps we should tweak the wording of the `it()` to make them a bit more descriptive. Currently, it'll just print: `Spec name: Admin disputes loads disputes, description: loads disputes` when the test is run.

What do you think about making the `it` value something like `page should load without any errors.`?

Reply:
💯 Agree, I need to think better tests descriptions next time.